virttest/utils_sriov.py

```diff
@@ -86,6 +86,40 @@ def get_pf_info(session=None):
     return pf_info
 
 
+def get_pf_pci(session=None):
+    """
+    Get the pci id of the available PF
```

Comment:
It would be more clear if update the description more detailed, like:
Get the pci id(eg. 0000:05:10.1) of the available(status='up') PF

Reply:
I have ran these functions and works well.

Reply:
Updated.